server/config/auth/passportConfig.js

```diff
@@ -0,0 +1,29 @@
+import passport from 'passport';
+import passportJWT from 'passport-jwt';
+import users from '../../db/controllers/users';
+
+const ExtractJwt = passportJWT.ExtractJwt;
+const JwtStrategy = passportJWT.Strategy;
+
+const jwtOptions = {};
+jwtOptions.jwtFromRequest = ExtractJwt.fromAuthHeader();
+jwtOptions.secretOrKey = 'JjS6E32hy1yBbhFN7uYI8t8IqpPy7xgw';
+
+const strategy = new JwtStrategy(jwtOptions, (jwtPayload, next) => {
+  console.log('payload received', jwtPayload);
```

Comment:
I know I told you this but for documentation's benefit I'm keeping console logs generally just because its useful for development, but we should remove them where it makes sense before finalizing our code.